app/api/views/tasks.py

```diff
@@ -0,0 +1,95 @@
+import sys
+import inspect
+from flask import jsonify, current_app, request
+from app.tasks.brief_tasks import process_closed_briefs
+from app.tasks.mailchimp import sync_mailchimp_seller_list, send_new_briefs_email
+from app.api import api
+from app.api.helpers import role_required
+
+
+@api.route('/tasks', methods=['GET'])
+@role_required('admin')
+def list_tasks():
+    """Lists triggerable tasks
+    ---
+    tags:
+      - tasks
+    responses:
+      200:
+        type: string
+        description: string
+    """
+    functions = inspect.getmembers(sys.modules[__name__], inspect.isfunction)
```

Comment:
perhaps this wasn't such a great idea - this code is complicated and I forgot the routes would be listed in swagger anyway

Reply:
its gone now